lib/Lifters/ValueLifter.cpp

```diff
@@ -142,22 +131,14 @@ static llvm::Constant *UnwrapZeroIndices(llvm::Constant *ret,
 llvm::Constant *
 ValueLifterImpl::TryGetPointerForAddress(uint64_t ea,
                                          EntityLifterImpl &ent_lifter,
-                                         llvm::PointerType *hinted_type) const {
+                                         llvm::Type *hinted_type) const {
```

Comment:
Gawd I wish that instead we refactored the value lifted into non-existence, and got it to use the `CrossReferenceResolver` somehow.

Reply:
Do want to do this in this PR? Or this more of a future "nice to have"?

Reply:
Nice to have, not this PR.